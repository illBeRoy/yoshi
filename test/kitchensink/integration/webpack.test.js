describe('something', () => {
  beforeEach(async () => {
    await page.goto('http://localhost:3000');
  });

  it('should', async () => {
    expect(await page.title()).toMatchSnapshot();
  });
});
