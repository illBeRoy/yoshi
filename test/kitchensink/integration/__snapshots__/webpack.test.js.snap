// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`something should 1`] = `"Wix Full Stack Project Boilerplate"`;
